pyHS100/smartbulb.py

```diff
@@ -78,6 +84,20 @@ def is_variable_color_temp(self) -> bool:
         """
         return bool(self.sys_info['is_variable_color_temp'])
 
+    @property
+    def valid_temperature_range(self) -> Tuple[int, int]:
+        """
+        Returns the white temperature range (in Kelvin)
+        depending on the bulb model
+
+        :return: White temperature range in Kelvin (minimun, maximum)
+        :rtype: tuple
+        """
+        for model, temp_range in TPLINK_KELVIN.items():
+            if re.match(model, self.sys_info['model']):
+                return temp_range
+        return False
```

Comment:
Please return (0, 0) instead of False (as `-> Tuple[int, int]` waits for a tuple and it's better not to make confusion between types).

Reply:
ok !